samples/sensor/ti_hdc/README.rst

```diff
@@ -0,0 +1,81 @@
+.. _ti_hdc_sample:
+
+TI_HDC Sample
+##############
+
+Description
+***********
+
+This sample application periodically takes Temp/RH measurements
+using the TI_HDC sensor. The result is written to the console.
+
+Requirements
+************
+
+This sample needs an TI_HDC sensor connector to the target
```

Comment:
There is no TI_HDC sensor, "sample needs a compatible sensor like HDC1010, HDC1008"

Reply:
Noted